Jenkinsfile.promote

```
@@ -83,11 +94,44 @@ pipeline {
                 }
             }
         }
-    }
-    post {
-        always {
-            script {
-                cleanImages()
+        stage('Merge PR')
+        {
+            when {
+                expression {
+                    return isRelease() && params.DEPLOY_BUILD_URL != ''
+                }
+            }
+            steps{
+                script {
+                    githubscm.mergePR(deployProperties['kogito-images.pr.link'])
+                }
+            }
+        }
+        stage('Git Tag')
+        {
+            when {
+                return isRelease() && params.DEPLOY_BUILD_URL != ''
+            }
+            steps {
+                script {
+                    checkout(githubscm.resolveRepository("kogito-images", params.GIT_AUTHOR, params.BUILD_BRANCH_NAME, false))
+                    def tagName = getGitTag()
+                    githubscm.tagRepository( tagName, env.BUILD_TAG)
```

No need for a variable here:

```suggestion
                    githubscm.tagRepository( getGitTag(), env.BUILD_TAG)
```